docs/dev/shims.md

```diff
@@ -17,7 +17,7 @@ common code, maximize reuse, and minimize logic duplication.
 This is achieved by using a ServiceProvider pattern. All Shims implement the same API,
```

Comment:
Are there any ramifications of the new shimplify setup of sources for setting breakpoints in the IDEs in shim-shared code?

Reply:
added some language for IDEA, could not verify the state of the art for VS Code due to #7767